packages/suite-web-landing/pages/_document.tsx

```diff
@@ -39,6 +39,38 @@ export default class MyDocument extends Document {
         return (
             <Html lang="en" style={{ height: '100%' }}>
                 <Head>
+                    <meta name="title" content="Trezor Suite" />
+                    <meta
+                        name="description"
+                        content="New desktop &amp; browser app for Trezor hardware wallets. Trezor Suite brings big improvements across our three key pillars of usability, security and privacy."
+                    />
+
+                    {/* Open Graph / Facebook */}
+                    <meta property="og:type" content="website" />
+                    <meta property="og:url" content="https://metatags.io/" />
+                    <meta property="og:title" content="Trezor Suite" />
+                    <meta
+                        property="og:description"
+                        content="New desktop &amp; browser app for Trezor hardware wallets. Trezor Suite brings big improvements across our three key pillars of usability, security and privacy."
+                    />
+                    <meta
+                        property="og:image"
+                        content="https://metatags.io/assets/meta-tags-16a33a6a8531e519cc0936fbba0ad904e52d35f34a46c97a2c9f6f7dd7d336f2.png"
+                    />
+
+                    {/* Twitter */}
+                    <meta property="twitter:card" content="summary_large_image" />
+                    <meta property="twitter:url" content="https://metatags.io/" />
+                    <meta property="twitter:title" content="Trezor Suite" />
+                    <meta
+                        property="twitter:description"
+                        content="New desktop &amp; browser app for Trezor hardware wallets. Trezor Suite brings major improvements across our three key pillars of usability, security and privacy."
+                    />
+                    <meta
+                        property="twitter:image"
+                        content="https://metatags.io/assets/meta-tags-16a33a6a8531e519cc0936fbba0ad904e52d35f34a46c97a2c9f6f7dd7d336f2.png"
```

Comment:
replace with local copy of https://user-images.githubusercontent.com/29627086/97498925-4eda6f80-196d-11eb-8f54-ae8cfe76ce62.png